tests/ovn-northd.at

```diff
@@ -10907,3 +10907,77 @@ CHECK_NO_CHANGE_AFTER_RECOMPUTE
 
 AT_CLEANUP
 ])
+
+OVN_FOR_EACH_NORTHD_NO_HV([
```

Comment:
It might be good to also add a test that runs `ovn-trace` and checks that packets are sent to conntrack when the option is enabled.  A similar example is the "ACL allow-stateless omit conntrack - Logical_Switch" test.